bin/hdfcoinc/pycbc_multiifo_add_statmap

```diff
@@ -0,0 +1,214 @@
+#!/bin/env python
+""" Reduce IFAR for coincidences in times with more than one ifo combination
+available. Cluster for best IFAR. This clusters to find the most
+significant foreground, but leaves the background triggers alone.
```

Comment:
'Reduce IFAR' -> 'Calculate total FAR based on statistic ranking' 
'Cluster to keep coincs with highest statistic'